README.md

```diff
@@ -7,15 +7,15 @@ It includes a pre-defined set of modules and classes that are essentially wrappe
 
 From the command line:
 ```bash
-gem install justifi --version "0.3.0" --source "https://rubygems.pkg.github.com/justifi-tech"
+gem install justifi --version "0.3.1" --source "https://rubygems.pkg.github.com/justifi-tech"
 ```
 OR
 
 Add these lines to your application's Gemfile:
 
 ```ruby
 source "https://rubygems.pkg.github.com/justifi-tech" do
-  gem "justifi", "0.3.0"
+  gem "justifi", "0.3.1"
```

Comment:
It's not gonna break past generated packages, but if anyone updates when we generate a new package it will.

But since this would make it go to `1.0 `, I didn't want to generate a new package just yet because we still have some uncovered API resources like refunds and payouts. I was thinking we could release `1.0 ` with those resources too.

That's what I had in mind but I'm okay with a new package too.

Reply:
Ah, good to know. I haven't worked on ruby gems yet and wasn't aware that you had to generate a new package at a certain point. I totally agree with your plan to upgrade to 1.0 once the missing  API resources are added. Thank you for explaining!